amp-camp/src/server/server.js

```diff
@@ -312,4 +331,22 @@ function enableCors(req, res) {
     res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
     res.header("Access-Control-Expose-Headers", "AMP-Access-Control-Allow-Source-Origin,AMP-Redirect-To");
     res.header("AMP-Access-Control-Allow-Source-Origin", sourceOrigin);
+}
+
+// Uses mustache to render the dynamic page. Injects canonical tag, based on the requested URL.
+function renderPage(req, res, template, responseJsonObj) {
```

Comment:
I don't understand why this is called `renderPage()`.

Reply:
I think we also need to clearly separate these helper functions from the main express.js routing. They could be in their own file or their own class, but I think I'd be ok with just a large comment with nice stars in it separating these from the rest :)

Reply:
I called it renderPage(), because it's the only one calling the res.render().
Do you think there's any other better name for this?

Reply:
Agreed with moving helper functions somewhere else. I think that's the next task to do (refactor).

Reply:
Ok! We can cover the rest of this in this week's refactor... lgtm!